src/redux/selectors/graphCardSelectors.js

```diff
@@ -7,39 +7,27 @@ import { reduxHelpers } from '../common/reduxHelpers';
 
 const graphCardCache = { dataId: null, data: {} };
 
-const graphComponent = (state, props = {}) => ({ ..._get(state, ['graph', 'component', props.viewId]) });
-
 const graphResponse = (state, props = {}) => ({
   ..._get(state, ['graph', 'reportCapacity', props.productId]),
-  ...{ viewId: props.viewId, productId: props.productId }
+  ...{ viewId: props.viewId, productId: props.productId, graphGranularity: props.graphGranularity }
```

Comment:
Optional: May be able to get away with writing... there may however be additional unintended props
```
const graphResponse = (state, props = {}) => ({
   ...props,
   ..._get(state, ['graph', 'reportCapacity', props.productId])
});
```

Reply:
Rolling with existing format for now, keeping this in mind for the future.